src/libraries/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableDictionary.cs

```diff
@@ -336,13 +337,13 @@ public static class ImmutableDictionary
         /// The value for the key, or <paramref name="defaultValue"/> if no matching key was found.
         /// </returns>
         [Pure]
-        public static TValue GetValueOrDefault<TKey, TValue>(this IImmutableDictionary<TKey, TValue> dictionary, TKey key, TValue defaultValue)
+        public static TValue GetValueOrDefault<TKey, TValue>(this IImmutableDictionary<TKey, TValue> dictionary, TKey key, TValue defaultValue) where TKey : notnull
```

Comment:
As defaultValue value is provided from caller don't need to put `[return: MaybeNull]`

Reply:
Correct.